node/src/accountant/mod.rs

```diff
@@ -4851,13 +4905,70 @@ mod tests {
 
         System::current().stop();
         system.run();
-        let fingerprints_rowids_params = fingerprints_rowids_params_arc.lock().unwrap();
-        assert_eq!(*fingerprints_rowids_params, vec![vec![expected_hash]]);
-        let mark_pending_payables_rowids_params =
-            mark_pending_payables_rowids_params_arc.lock().unwrap();
+        let inserted_new_records_params = inserted_new_records_params_arc.lock().unwrap();
         assert_eq!(
-            *mark_pending_payables_rowids_params,
-            vec![vec![(expected_wallet, expected_rowid)]]
+            inserted_new_records_params[0],
+            BTreeSet::from([expected_tx])
+        );
+        let pending_payable_notify_later_params =
+            pending_payable_notify_later_params_arc.lock().unwrap();
+        assert_eq!(
+            *pending_payable_notify_later_params,
+            vec![(ScanForPendingPayables::default(), pending_payable_interval)]
+        );
+        // The accountant is unbound here. We don't use the bind message. It means we can prove
+        // none of those other scan requests could have been sent (especially ScanForNewPayables,
+        // ScanForRetryPayables)
+    }
+
+    #[test]
+    fn accountant_processes_sent_payables_with_retry_and_schedules_pending_payable_scanner() {
+        let mark_pending_payables_rowids_params_arc = Arc::new(Mutex::new(vec![]));
+        let pending_payable_notify_later_params_arc = Arc::new(Mutex::new(vec![]));
+        let inserted_new_records_params_arc = Arc::new(Mutex::new(vec![]));
+        let expected_wallet = make_wallet("paying_you");
+        let expected_hash = H256::from("transaction_hash".keccak256());
+        let payable_dao = PayableDaoMock::new()
+            .mark_pending_payables_rowids_params(&mark_pending_payables_rowids_params_arc)
+            .mark_pending_payables_rowids_result(Ok(()));
```

Comment:
I am surprised that this is still an active item in the code. 

Didn't we agree that `mark_pending_payables` will go away? It should not be used in the production code anymore.

Don't remove it just here, it first must be cut out from the place where the referenced fn is called.